Conference-Android/app/src/main/res/layout/conference_list_view_item.xml

```diff
@@ -1,19 +1,50 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<android.support.v7.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:card_view="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/card_view"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:focusable="true"
-    android:clickable="true">
+    android:layout_gravity="center"
+    android:layout_margin="20dp"
+    android:clickable="true"
+    card_view:cardCornerRadius="4dp"
+    card_view:cardElevation="30dp">
 
-    <TextView
-        android:id="@+id/name"
+    <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+        android:id="@+id/conferenceLayout"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_marginTop="10dp"
-        android:layout_marginLeft="5dp"
-        android:layout_marginRight="5dp"
-        android:text="Volley"
-        android:textSize="18sp"
-        />
+        android:layout_height="match_parent"
+        android:clickable="true"
+        android:focusable="true"
+        android:orientation="vertical">
 
-</LinearLayout>
+        <ImageView
+            android:id="@+id/conferenceImage"
+            android:layout_width="512px"
+            android:layout_height="512px"
```

Comment:
Kind of inline with the "no magic numbers" rule, I'd highly recommend using `dimens` for all values. I'd also recommend against using `px` since it won't adjust for different screen densities and can lead to unexpected layouts.

My projects tend to have something like:

```
@dimen/layout_padding = 16dp
@dimen/layout_padding_half = 8dp
@dimen/layout_padding_quarter = 4dp
# and so on...
```

And then restricting myself to using only these sizes results in a much more consistent layout throughout the app.